listenbrainz/webserver/static/js/tests/utils/utils.test.tsx

```diff
@@ -137,4 +138,42 @@ describe("searchForSpotifyTrack", () => {
       searchForSpotifyTrack("foobar", "import", "vs", "star")
     ).resolves.toEqual(spotifySearchResponse.tracks.items[0]);
   });
+
+  it("retries if network error / submit fails", async () => {
+    // Overide mock for fetch:
+    window.fetch = jest.fn().mockImplementation(() => {
+      // 1st call will recieve a network error
+      return Promise.reject(new Error("Oh no!"));
+    });
+    await expect(
+      searchForSpotifyTrack("foobar", "import", "vs", "star")
+    ).rejects.toThrow(Error("Oh no!"));
+  });
+
+  it("skips if any other response code is recieved", async () => {
```

Comment:
I think the code in utils.tsx is wrong or has some issues at line number L: 46 to 48, it is trying to throw string directly which is probably wrong. That's why this test is failing. Please review this!

Reply:
You're right @vardansaini , it should be `throw new Error(responseBody.error);`. Can you please make the required modification?

Reply:
sure!

Reply:
done